kvmagent/kvmagent/plugins/nvram/nvram_local.py

```
@@ -0,0 +1,171 @@
+        if not os.path.exists(mount_path):
+            bash.bash_r("mkdir -p %s" % mount_path)
+
+        is_mounted = False
+        with open('/proc/mounts', 'r') as f:
+            if any(mount_path in line for line in f):
+                is_mounted = True
+        
+        if not is_mounted:
+            ret = bash.bash_r("mount -t ext4 %s %s" % (loop_device_path, mount_path))
+            if ret != 0:
+                raise Exception("failed to mount %s to %s" % (loop_device_path, mount_path))
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)